### DISCUSSION TITLE
转为的overlaybd镜像无法使用，帮忙看看是哪个步骤不对

### DISCUSSION THREAD

**Discussion by jinteng123** [containerd/overlaybd#280] [General]
使用这个命令进行转换
 /opt/overlaybd/snapshotter/ctr obdconv --fstype "xfs" docker.io/library/redis:6.2.1 localhost:5000/redis:6.2.1_obd_xfs
 转为后发现镜像size 为0
 
![image](https://github.com/containerd/overlaybd/assets/22409094/1af0e49c-6535-4ae3-a35c-5785e1fbfd1a)

nerdctl run --snapshotter=overlaybd localhost:5000/redis:6.2.1_obd_xfs
运行返回这个错误
FATA[0000] failed to prepare extraction snapshot "extract-325776645-qJ5d sha256:c269cf47609c3e3e116f3c3dd4ac6d864c52d36f672f001fb81c0f191c61ee73": failed to attach and mount for snapshot 81262: failed to enable target for /sys/kernel/config/target/core/user_999999999/dev_81262, failed:empty repoBlobUrl: unknown

**Comment by WaberZhuang** [comment]
试试将`localhost:5000/redis:6.2.1_obd_xfs`先上传至仓库再拉下来使用。
obdconv转换得到的镜像需要先推送到仓库再拉下来，暂时不支持本地转换的镜像直接使用，会有意料之外的各种问题。

**Reply by WaberZhuang** [reply]
EN:
Try uploading `localhost:5000/redis:6.2.1_obd_xfs` to the repository and pulling it instead of directly using the locally converted image. Locally converted images are not supported for direct use at this time, or unexpected problems may occur.

**Reply by jinteng123** [reply]
这个已经可以了，还有一个问题，我想转为成turboOCI这个格式的类型， /opt/overlaybd/snapshotter/convertor -r sea.hub:5000/library -i nginx:1.25.1 --turboOCI nginx:1.25.1_turbo --verbose，仓库用的是自签的证书
![image](https://github.com/containerd/overlaybd/assets/22409094/3b255631-506c-4b16-9d9b-05ec4bdb0db9)

报日志是证书不可信任，这个得怎么解决，谢谢

**Reply by WaberZhuang** [reply]
Try to pull commit [#237](https://github.com/containerd/accelerated-container-image/pull/237) (the latest version of accelerated-container-image), which supports certification for convertor.

##### Usage
```
# Self-signed certificates may require option '--insecure'
--client-cert ${cert-file}:${key-file} --insecure
```

If it still fails, please let me know.

Besides, if there are any new issues with image conversion, please go to [accelerated-container-image](https://github.com/containerd/accelerated-container-image) and open a new issue in English, so it's easier for the community to refer to it.
